pandas/tests/io/excel/test_openpyxl.py

```diff
@@ -1,124 +1,123 @@
 import pytest
 
-import pandas.util._test_decorators as td
-
 from pandas import DataFrame
 from pandas.util.testing import ensure_clean
 
 from pandas.io.excel import ExcelWriter, _OpenpyxlWriter
 
-
-@td.skip_if_no('openpyxl')
-@pytest.mark.parametrize("ext", ['.xlsx'])
-class TestOpenpyxlTests:
-
-    def test_to_excel_styleconverter(self, ext):
-        from openpyxl import styles
-
-        hstyle = {
-            "font": {
-                "color": '00FF0000',
-                "bold": True,
-            },
-            "borders": {
-                "top": "thin",
-                "right": "thin",
-                "bottom": "thin",
-                "left": "thin",
-            },
-            "alignment": {
-                "horizontal": "center",
-                "vertical": "top",
-            },
-            "fill": {
-                "patternType": 'solid',
-                'fgColor': {
-                    'rgb': '006666FF',
-                    'tint': 0.3,
-                },
-            },
-            "number_format": {
-                "format_code": "0.00"
-            },
-            "protection": {
-                "locked": True,
-                "hidden": False,
+openpyxl = pytest.importorskip("openpyxl")
+
+pytestmark = pytest.mark.parametrize("ext", ['.xlsx'])
```

Comment:
isn't this just a fixture definition?

Reply:
do we want more fixtures?

we could have a fixture and inject into all tests or we could alias this mark and decorate all tests.

this just applies the parametrisation to all tests in module automatically.

Reply:
Not yet but we should probably create a `write_ext` fixture for use here and in the writer tests (the existing fixture is applicable to extensions that we read)

Reply:
i just find this very opaque here (we don't use this idiom anywhere else)

Reply:
Probably better done in a separate PR IMO as that will require a decent amount of updates